src/main/kotlin/it/pagopa/wallet/services/WalletService.kt

```diff
@@ -1003,11 +1003,27 @@ class WalletService(
                         "Unhandled wallet details variant in getAuthData for wallet id ${wallet.id}")
             }
 
+        val contextualOnboardDetails =
```

Comment:
please add junit tests to cover cases where `contextualOnboardDetails` are present and cases where are not based on wallet metadata info